client/src/components/Map/Map.js

```diff
@@ -99,11 +105,11 @@ export default function Map() {
   if (!isLoaded) return "Loading Maps";
 
   return (
-    <div>
-      <MapHeader>
+    <>
+      {/* <MapHeader>
         <Search panTo={panTo} />
         <Locate panTo={panTo} />
-      </MapHeader>
+      </MapHeader> */}
```

Comment:
If you don't need it, consider to remove it :)

Reply:
I´m still working on it. I want to ```Search panTo``` and ```Locate panTo```to be integrated in the map.

Reply:
Opened a new issue for it an will merge this branch